registrants/src/API/EMBC.Registrants.API/Mappers/Mappings.cs

```diff
@@ -149,6 +150,7 @@ public Mappings()
                         d.RecipientLastName = eTransfer.RecipientLastName;
                         d.SecurityQuestion = eTransfer.SecurityQuestion;
                         d.SecurityAnswer = eTransfer.SecurityAnswer;
+                        d.IsSelfServe = s.IsSelfServe;
```

Comment:
I don't think this needs to be in the aftermap, it can be mapped directly above instead of ignoring it